tests/interpret/input_reduction_test.py

```diff
@@ -25,22 +25,3 @@ def test_input_reduction(self):
         )  # input reduction removes tokens
         for word in reduced["final"][0]:  # no new words entered
             assert word in reduced["original"]
-
-        # test using NER model (tests different underlying logic)
```

Comment:
I removed this test due to the same reason described [here](https://github.com/allenai/allennlp/pull/4924/files#r563158779).